haskell/private/actions/link.bzl

```diff
@@ -124,6 +124,13 @@ def link_binary(
 
     args.add(["-o", compile_output.path, dummy_static_lib.path])
 
+    # In compile.bzl, we pass this just before all -package-id
```

Comment:
This comment should be copy pasted in front of all similar lines. This will motivate us to factorise and document more.

Reply:
Added a reference. Duplication in comments is worse than in code, because comments are not checked.